### PR TITLE
Homework

### PR DESCRIPTION
**Pull request by franzferdinnand**

### CODE REVIEW

File: memory_usage.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import requests
+import functools
+import psutil
+
+
+def func_memory(func):
+    def get_process_memory():
+        process = psutil.Process()
```
Comment (mikolaz27) [inline memory_usage.py:8]: looks strange.  remove this function and change to this
no reason to create function for 2 rows

process = psutil.Process()
start_memo = process.memory_info().rss 
<legacy function>
end_memo = process.memory_info().rss

Comment (franzferdinnand) [reply memory_usage.py]: Done

Comment (mikolaz27) [reply memory_usage.py]: What about tracemalloc?

Comment (franzferdinnand) [reply memory_usage.py]: Знімок екрана 2022-11-16 о 21 48 12

Comment (franzferdinnand) [reply memory_usage.py]: can't install it

Comment (franzferdinnand) [reply memory_usage.py]: need time to learn it

Comment (franzferdinnand) [reply memory_usage.py]: please check commit file memory_tracemalloc - is this correct solution?

Comment (mikolaz27) [reply memory_usage.py]: send it to the LMS

**Pull request merged**